scripts/xtensa-build-zephyr.py

```diff
@@ -120,6 +120,18 @@ class PlatformConfig:
 		"ace10_LX7HiFi4_2022_10",
 		ipc4 = True
 	),
+	"ptl" : PlatformConfig(
+		"intel", "intel_adsp/ace30_ptl",
+		f"RI-2022.10{xtensa_tools_version_postfix}",
+		"ace30_LX7HiFi4_PIF",
+		ipc4 = True
+	),
+	"ptl-sim" : PlatformConfig(
+		"intel", "intel_adsp/ace30_ptl_sim",
+		f"RI-2021.8{xtensa_tools_version_postfix}",
+		"ace30_LX7HiFi4",
```

Comment:
Pretty confused about what's happening here?  This string is an XCHAL_CORE_ID value, it's set at synthesis time when Cadence generates the SDK tarball and it is (should be) identical with the hardware all the way from pre-silicon validation through final software builds.  The fact the you have a different (!!?!) device instantiated for "sim" seems like wires got crossed somewhere?  That's not the way Cadence's stuff works.  Can you check with the hardware folks to see why you can't be simulating with the same device they did/will put on the chip?